be/src/Project/Services/Inventory/Inventory.Api/Services/InventoryItems/InventoryService.cs

```diff
@@ -475,5 +475,15 @@ public async Task UpdateInventoryQuantity(Guid? productId, double bookedQuantity
                 await _context.SaveChangesAsync();
             }
         }
+
+        public IEnumerable<string> GetInventorySuggestions(GetInventorySuggestionsServiceModel model)
+        {
+            var inventoryItems = _context.Inventory.Where(x => x.IsActive)
+                    .Include(x => x.Product)
+                    .AsSingleQuery()
+                    .Select(y => y.Product.Name);
+
+            return inventoryItems.Where(x => x.StartsWith(model.SearchTerm)).Take(model.Size);
```

Comment:
**suggestion** Please check if model.SearchTerm isNullOrWhiteSpace is false before return